### PR TITLE
Added tools/skeleton-dump tool

### PR DESCRIPTION
**Pull request by hyperlogic**
Debug tool that prints out the skeleton hierarchy of fbx files including joint indices, bindPose and defaultPoses.  The verbose option also prints the full FBX transformation set, pre/post rotations etc.

### CODE REVIEW

File: tools/skeleton-dump/src/SkeletonDumpApp.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
+//  VHACDUtil.h
```
Comment (ZappoMan) [inline tools/skeleton-dump/src/SkeletonDumpApp.h]: typo?

**Comment by hyperlogic** [comment]
Advanced User QA Test:
   * Build hifi from source
   * copy the build/tools/skeleton-dump/Release/skeleton-dump.exe into the build/interface/Release directory
   * open a command prompt into the build/interface/Release directory
   * run the following command `/skeleton-dump.exe -vi ../../../interface/resources/meshes/being_of_light/being_of_light.fbx`
   * It should print out the skeleton hierarchy for the default avatar

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1732448d3c3f6bd443ec17a660d995b36d0634b6 available at: https://gist.github.com/79ca91743a6b560fbefb06edcdf988ef<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-1732448.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-1732448.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 69fc048e5ee48571d4aee2f1c84b7ad985a97e9d available at: https://gist.github.com/79ca91743a6b560fbefb06edcdf988ef<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-69fc048.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-69fc048.dmg

**Comment by hyperlogic** [comment]
I have no clue why the windows build failed. test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 69fc048e5ee48571d4aee2f1c84b7ad985a97e9d available at: https://gist.github.com/79ca91743a6b560fbefb06edcdf988ef<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-69fc048.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9003-69fc048.dmg

**Comment by OmegaHeron** [comment]
Tested Ubuntu Linux 16.04

QA Pass

**Pull request merged**